tests/foreman/cli/test_abrt.py

```diff
@@ -76,7 +76,7 @@ def test_abrt_4(self):
 
         """
 
-    def test_abrt_5(self):
+    def test_positive_cli_report(self):
```

Comment:
To be honest, I have no idea about it since the test cases are not implemented and have very less info now.